code/modules/modular_computers/NTNet/_network.dm

```diff
@@ -98,8 +98,12 @@ GLOBAL_LIST_EMPTY(computer_networks)
 	return ARE_Z_CONNECTED(get_z(router.holder), get_z(D.holder))
 
 /datum/computer_network/proc/get_signal_strength(datum/extension/network_device/D)
-	if(check_connection(D))
-		return D.connection_type
+	if(!check_connection(D))
+		return 0
+	var/broadcast_strength = router.get_broadcast_strength()
+	var/distance = get_dist(router.holder, D.holder)
```

Comment:
I think it's better do get_turf() on both because not sure what get_dist would do to nested things (e.g. network card is holder, but its inside a PDA carried by a mob)

Reply:
it is done